cmd/client-admin.go

```diff
@@ -67,16 +67,14 @@ func newAdminFactory() func(config *Config) (*madmin.AdminClient, *probe.Error)
 				return nil, probe.NewError(e)
 			}
 
-			transport := http.DefaultTransport
-
-			if useTLS {
-				tlsConfig := &tls.Config{RootCAs: globalRootCAs}
-				if config.Insecure {
-					tlsConfig.InsecureSkipVerify = true
-				}
-				transport = &http.Transport{
-					TLSClientConfig: tlsConfig,
-				}
+			// Keep TLS config.
+			tlsConfig := &tls.Config{RootCAs: globalRootCAs}
+			if config.Insecure {
+				tlsConfig.InsecureSkipVerify = true
+			}
+
+			var transport http.RoundTripper = &http.Transport{
```

Comment:
This way, only TLSClientConfig is set but all other http.Transport fields are nil, which is not so good for us, can you use default values from http.DefaultTransport (https://golang.org/pkg/net/http/#RoundTripper)